core/src/scss/elements/table/_table.scss

```diff
@@ -14,37 +14,45 @@
 // Style guide: Elements.Table
 //
 table {
-  @include margin(2em 0);
+  @include margin(null 0);
+  @include modular-spacing('margin-bottom', 5);
+  border-collapse: collapse;
   border-spacing: 0;
   min-width: 100%;
 
-  th,
-  td {
-    @include padding(1.5rem);
-    border: 1px solid $su-color-driftwood;
+  caption {
+    @include caption;
   }
 
-  th {
-    background-color: $su-color-cloud;
-    color: $su-color-black;
-    font-weight: $su-font-regular;
-    text-align: left;
-  }
-
-  td {
-    background-color: $su-color-white;
-  }
+  thead,
+  tbody {
 
-  thead {
     th,
     td {
-      background-color: $su-color-cloud;
+      @include padding(1.5rem);
+
+      &:last-child {
```

Comment:
Yeah...looks like this need a little fixup. Right now it's targeting the last `td` or `th` and setting the margin-bottom to 0, what we really want is to target the last element within each `td` or `th`.